.gitignore

```diff
@@ -39,3 +39,9 @@ paket-files/
 .idea/
 
 deploy/
+
+# Ionide (cross platform F# VS Code tools) working folder
+.ionide/
+
+# VSCode working folder
+.vscode/
```

Comment:
Depending on your preferences you might tweak this into:

```
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
*.code-workspace

# Local History for Visual Studio Code
.history/
```

Reply:
Sure, you can include that as well.